### ISSUE TITLE
Suggestion for docs about the available recipients list

### ISSUE THREAD

**Issue by vicrec** [michael-milette/moodle-local_contact#35]
Hello, at first, thanks for your work! This plugin saved me a lot of time.
I got a suggestion about documentation. It could be specified, more strictly, that the lines of avaiable recipient list must have no whitespace char before and after the `|` char. I got some email errors:
````
Can not send email to user without email: -99
    line 5726 of /lib/moodlelib.php: call to debugging()
    line 312 of /local/contact/class/local_contact.php: call to email_to_user()
    line 111 of /local/contact/index.php: call to local_contact->sendmessage()
````
After 1h debugging the code, i realized that it was caused by a whitespace char inside the line:

`tech support| john@example.com| John Doe`

So i removed the illegal chars in this way:

`tech support|john@example.com|John Doe`

and all worked fine.

I hope it will help!

**Comment by michael-milette** [comment]
Hi @vicrec ,

You are very welcome. I am always gratified to hear that people are getting good use from my work.

Thank you very much for that suggestions. I will add it to the documentation. However, I will also be taking it one step further by updating the code so that extra spaces will no longer cause this problem.

Thank you again for taking the time to write and your excellent suggestion. It is because of to people like you that I become a better developer and produce better quality products.

Best regards,

Michael

**Comment by michael-milette** [comment]
HI @vicrec 

This has been fixed by fix #36 . The updated version is only available on GitHub right now as I haven't had time to fully test it. Let me know if you have any problems with it.

Best regards,

Michael
